cloud/create-databases.yml

```diff
@@ -0,0 +1,19 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+    - "{{ local_secret_dir }}/{{ cluster }}/cluster-secrets.yml"
```

Comment:
I'm not sure if it's possible in Ansible but it would be great if we could have a vars file that accepts `cluster` as a parameter and includes all vars, constants and secrets for that cluster. This would reduce this to a single line.

@michalpiasecki1 Do you know any mechanism that would allow us to do this in a clean way?